### PR TITLE
images: implemented image resizing

### LINKED ISSUE #6: resize uploaded images
In the current implementation, the images are stored in the same size as they are uploaded. This would result in much higher memory consumption. It would be more efficient if the images are compressed before storing.

### PR DESCRIPTION
**Pull request by FaiyazMujawar**
Fixes #6 

In this PR,
- implemented functionality to resize images before saving on the disk.
- The images are resized to default size of 1366x720p

### CODE REVIEW

File: routes/file.js [hunk 1]
```diff
@@ -1,15 +1,46 @@
 const router = require("express").Router();
+const sharp = require("sharp");
+const { writeFileSync } = require("fs");
+
 const saveImages = require("../services/MulterService");
 
+/** Size to which the image is to be resized */
+const IMAGE_SIZE = {
+  height: 1080,
```
Comment (drigio) [inline routes/file.js]: Make height `1366`

File: routes/file.js [hunk 2]
```diff
@@ -1,15 +1,46 @@
...
-  });
+
+  try {
+    files.forEach(async (file) => {
+      const { path } = file;
+
+      // Resizing image
+      const buffer = await sharp(path)
+        .resize(IMAGE_SIZE.height, IMAGE_SIZE.width)
+        .jpeg({ quality: 90 })
```
Comment (drigio) [inline routes/file.js]: reduce quality even more. We want less size of file.

Comment (FaiyazMujawar) [reply routes/file.js]: Specify a value for quality

Comment (drigio) [reply routes/file.js]: Make it 60-70 maybe

**Comment by drigio** [comment]
LGTM! Merging in.

**Pull request merged**